pkg/containers/runtime/sockets.go

```diff
@@ -0,0 +1,34 @@
+package runtime
```

Comment:
containersruntime?

Reply:
I personally find that name too long. 
I had a conversation with Yaniv where I was considering `cri` but we agreed that may mislead to think this relates to the k8s cri (although there are similarities).